Packs/Ncurion/Integrations/Ncurion/Ncurion.py

```diff
@@ -0,0 +1,198 @@
+    return log_list
+
+
+def raw_response_to_context_rules(items: Union[Dict, List]) -> Union[Dict, List]:
+    if isinstance(items, list):
+        return [raw_response_to_context_rules(item) for item in items]
+    return {
+        'Id': items.get('id'),
+        'Name': items.get('name'),
+        'Host': items.get('host'),
+        'Log_storage_month_period': items.get('log_storage_month_period'),
+        'Useno': items.get('useno'),
+        'Sync_state': items.get('sync_state'),
+        'Is_connected': items.get('is_connected'),
+        'Description': items.get('description'),
+        'Created_at': items.get('created_at'),
+        'Updated_at': items.get('updated_at')
+    }
+
+
+def switch_list_to_list_counter(data: Union[Dict, List]) -> Union[Dict, List]:
+    if isinstance(data, list):
+        return [switch_list_to_list_counter(dat) for dat in data]
+    new_data = {}
+    for item in data:
+        if type(data[item]) == list:
+            new_data[item] = len(data[item])
+        elif data[item] and type(data[item]) == dict:
+            counter = 0
+            for in_item in data[item]:
+                if type(data[item][in_item]) == list:
+                    counter += len(data[item][in_item])
+                elif data[item][in_item]:
+                    counter = 1 if counter == 0 else counter
+            new_data[item] = counter
+        else:
+            new_data[item] = data[item]
+    return new_data
+
+
+def get_log_list(base_url, username, password):
+    access_token, refresh_token, headers1 = login(base_url, username, password)
+    logserver_url = base_url + '/logapi/api/v2/logmgr/0'
+    verify_certificate = not demisto.params().get('insecure', False)
+    log_list = requests.request("GET", logserver_url, headers=headers1, verify=verify_certificate)
+    data = log_list.text
+    items = json.loads(data)
+    context_entry = raw_response_to_context_rules(items)
+    api_log_out(base_url, access_token, refresh_token, headers1)
+    results = CommandResults(
+        outputs_prefix=f'{INTEGRATION_CONTEXT_NAME}.logsever',
+        outputs_key_field='id',
+        outputs=context_entry
+    )
+    return_results(results)
+
+
+def fetch_incidents(base_url, username, password, last_run: Dict[str, int], first_fetch_time: Optional[int])-> Tuple[Dict[str, int], List[dict]]:
+    access_token, refresh_token, headers1 = login(base_url, username, password)
+    log_list = loglist(base_url, access_token, refresh_token, headers1)
+    log_server_id = [e["id"] for e in log_list if e["is_connected"] == True]
+    last_fetch = last_run.get('last_fetch', None)
+
+    if len(log_server_id) == 0:
+        return_output('ok')
+    else:
+            max_fetch = demisto.params().get('max_fetch')
+            #incidents: List[Dict[str, Any]] = []
+            incidents = []
+            now_time = datetime.datetime.utcnow()
+            params1 = {"start": f"{last_fetch}", "end": f"{now_time}", "size": max_fetch}
```

Comment:
The api returns results including the "end" or not? 
Because if yes, you maybe get duplicate results.